scrnaseq_hto.Rmd

```diff
@@ -258,23 +444,40 @@ p = patchwork::wrap_plots(p_list, ncol = 2) +
   patchwork::plot_annotation("Normalised HTO data") + 
   patchwork::plot_layout(guides = "collect") & 
   theme(legend.position="bottom") 
-p
+suppressMessages(p)
 ```
 
-Lastly, we compare the number of features between classified cells. 
+Lastly, we compare the counts and number of features between classified cells. 
 ```{r hto_plot_features}
 # Number of features in the different cells
 nfeature_metrics = grep("_HTO", 
                         grep("nFeature_", colnames(sc[[]]), v=TRUE),
                         v=TRUE,
                         invert=TRUE)
-p = VlnPlot(sc, features=nfeature_metrics, idents=levels(Seurat::Idents(sc)), pt.size=0) + 
+ncounts_metrics = grep("_HTO", 
```

Comment:
Not quite sure what this is, but will see once I run the script.

Reply:
Shows counts and features for the HTOs